src/nbl/video/CVulkanImage.h

```diff
@@ -7,14 +7,13 @@
 
 #include <volk.h>
 #include "nbl/video/IGPUImage.h"
+#include "nbl/video/CVKLogicalDevice.h"
 
 namespace nbl
 {
 namespace video
 {
 
-class CVKLogicalDevice;
-
 class CVulkanImage final : public IGPUImage, public IDriverMemoryAllocation
```

Comment:
under Vulkan, do not inherit from `IDriverMemoryAllocation`

Vulkan has an actual object for a memory allocation